maestro/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
can be replaced with readiness or liveness probe
https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/

Reply:
there is a livness probe : https://github.com/overture-stack/helm-charts/pull/2/files#diff-6a3cfb9a25e467048b8ddfc3fd06a6f6R43

this is a helm hook to execute helm test https://helm.sh/docs/developing_charts/#the-available-hooks, exmaple:
https://github.com/helm/charts/blob/master/stable/nextcloud/templates/tests/test-connection.yaml